app/src/main/java/com/mument_android/app/di/RepositoryModule.kt

```diff
@@ -111,4 +114,11 @@ object RepositoryModule {
         mumentSummaryMapper: MumentSummaryMapper,
         mumentListDataSource: MumentListDataSource
     ): MumentListRepository = MumentListRepositoryImpl(mumentSummaryMapper, mumentListDataSource)
+
+    @Provides
+    @Singleton
+    fun provideBlockUserRepository(
+        blockUserDataSource: BlockUserDataSource,
+        blockUserMapper: BlockUserMapper
+    ): BlockUserRepository = BlockUserRepositoryImpl(blockUserDataSource, blockUserMapper)
```

Comment:
- Repository도 동일합니다!!
- 물론 해당 클래스에서 하는 것이 아닌 추상 클래스에서 해야합니당!!
```suggestion
    @Binds
    @Singleton
    abstract fun bindsBlockUserRepository(
        blockUserRepositoryImpl: BlockUserRepositoryImpl
    ): BlockUserRepository
```